docs_src/microservices/device/supported/device-usb-camera/walkthrough/custom-build.md

```diff
@@ -85,8 +85,26 @@ The hostname and port of the RTSP server can be configured in the `Driver` secti
     Driver:
         RtspServerHostName: "localhost"
         RtspTcpPort: "8554"
+        RtspAuthenticationServer: "localhost:8000"
```

Comment:
we can mention that this is the hostname and port that the device service will run the authentication server on